aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -127,12 +128,16 @@ Tensor empty_strided_cpu(IntArrayRef size, IntArrayRef stride, const TensorOptio
   return t;
 }
 
-Tensor& empty_out(Tensor& result, IntArrayRef size) {
+Tensor& empty_out(
+    Tensor& result,
+    IntArrayRef size,
+    MemoryFormat memory_format) {
   check_size_nonnegative(size);
   if (result.is_sparse()) {
```

Comment:
can we check that memory_format is sane if the tensor is sparse?

Reply:
Solved by making memory_format optional